inc/template-tags.php

```diff
@@ -157,3 +157,36 @@ function understrap_body_attributes() {
 		echo trim( $attributes ); // phpcs:ignore WordPress.Security.EscapeOutput
 	}
 }
+
+if ( ! function_exists( 'understrap_comment_navigation' ) ) {
+	/**
+	 * Displays the comment navigation.
+	 *
+	 * @param string $nav_id The ID of the comment navigation.
+	 */
+	function understrap_comment_navigation( $nav_id ) {
+		if ( get_comment_pages_count() <= 1 || false === get_option( 'page_comments' ) ) {
```

Comment:
Can we drop `|| false === get_option( 'page_comments' )`? AFAIK, `get_comment_pages_count() <= 1` will be evaluated first. And `get_comment_pages_count()` includes the following code
```php
    if ( ! get_option( 'page_comments' ) ) {
        return 1;
    }
```
Code reference: https://core.trac.wordpress.org/browser/tags/5.8/src/wp-includes/comment.php#L1034

So, if `get_option( 'page_comments' ) = false` `get_comment_pages_count()` will return 1 and the whole if statement evaluates to true. No?

Reply:
Yes. It looks to me like that condition is already baked in.

Reply:
If you want to push that change, I'll merge this into develop and try it out.

Reply:
if you want to push that change, I'll merge it into develop.